evennia/contrib/worldloader/README.md

```diff
@@ -0,0 +1,126 @@
+# Intro
+It is a tool for Evennia(http://github.com/evennia/evennia) which can load game world from CSV files.
+
+
+## Installation
+You need to install Evennia and create your game first. Evennia's wiki is here: http://github.com/evennia/evennia/wiki.
+
+Copy the folder ```/example_tutorial_world/worlddata``` to your game folder. The directory should look like
```

Comment:
I would think the example data could remain in the evennia/contrib folder? Why does it need to be copied to the game folder? Would it not be easier for the user to simply supply the path to the CSV file to load, wherever it is?